\documentclass[ucla,biomath,12pt,noaddrfooter,datefrom]{UC_letter}
% The optional arguments are: University, Dept, font size, footer type, return address type ("from").
% University options include: ucla, ucd, uci, and ucsc (the default).
% UCLA department options include: biomath, biostat, hg, and stats.
% Footer options are: addrfooter, addrfooterall, and noaddrfooter (the default).
% From options are: nofrom, datefrom, fullfrom, and shortfrom (the default).

%\from{Dr.\ Marc Suchard, M.D., Ph.D.}
%\phone{+1.310.825.7442}
%\fax{+1.310.794.5446}
%\email{msuchard@ucla.edu}

\usepackage{graphicx}
%\usepackage{setspace}
\usepackage{url}
\usepackage{rotating} 
\usepackage{amsmath}
\usepackage{amssymb}
\setlength\parskip{\bigskipamount} \setlength\parindent{0pt}
\usepackage{color}


\newcommand{\bibsection}[1]{}
\newcommand{\section}[1]{}
\newcommand{\newblock}{}

   \newenvironment{thebibliography}[1]%
      {References\begin{description}}{\end{description}}
   \newcommand{\htmlbibitem}[2]{\label{#2}\item[{[#1]}]}
\usepackage[authoryear]{natbib}

\newenvironment{reply}{$\triangleright$\bf}{$\triangleleft$}
\renewenvironment{quote}
               {\list{}{\rightmargin\leftmargin}%
                \item\relax\normalfont}
               {\endlist}

%\providecommand{\tabularnewline}{\\}
%\usepackage{pstricks} 

%\setlength{\textheight}{9.0in}


%\newcommand{\citep}[1]{(citation in manuscript)}
%\renewcommand{\cite}[1]{(citation in manuscript)}

%\input{Baele_revision.xtr} 
% contains edited text, generated by make-edits.tex

\begin{document}

% consistent edits between manuscript and rebuttal letter
\input{FMDV_AMERICA.xtr} 

\bibliographystyle{mbe}

\begin{letter}{
Dr. Santiago F. Elena \\
Editor-in-Chief \\
Virus Evolution
}

\opening{Dear Dr.~Elena,}

Thank you very much for the opportunity to revise our manuscript entitled ``Spatio-temporal Dynamics of Foot-and-Mouth Disease Virus in South America" (Manuscript ID VEVOLU-2015-010) for consideration in Virus Evolution. In providing our revision, we have carefully considered the helpful suggestions and critiques of yourself and three Reviewers.

Attached you will find a point-by-point response (bold) to all comments (normal text) we received. Significant changes to the manuscript find themselves in quotes.

%\closing{Yours sincerely,}
%\vspace{-0.9in}
Yours sincerely, \\
\par
\vspace*{-0.2in}
\includegraphics[scale=0.75]{Suchard_signature} \\
%\vspace{-0.1in}
Marc A.~Suchard, M.D., Ph.D. \\
Professor

\clearpage

%====================
\textbf{Editor-in-Chief}
%====================

Dear Mr. Carvalho,

Manuscript ID VEVOLU-2015-010 entitled ``Spatio-temporal Dynamics of Foot-and-Mouth Disease Virus in South America'' which you submitted to Virus Evolution, has been reviewed.  
The comments of the reviewer(s) are included at the foot of this letter.

The three reviewers have very diverse opinions, from rejection (2nd reviewer) to minor revisions (3rd reviewer), thus the decision cannot be other but to request you to perform a major revision.  
Therefore, I invite you to respond to the reviewer(s)' comments and revise your manuscript.  
Please notice that given the nature of these comments and of the required amount of modifications, the new version will be send out for a second round of review, most likely to the same reviewers.

Once again, thank you for submitting your manuscript to Virus Evolution and I look forward to receiving your revision.

\begin{reply}
We thank the Editor-in-Chief and agree that the Reviewers' comments have helped us improve our manuscript. 
\end{reply}

Sincerely,
Dr. Santiago Elena
Editor-in-Chief
Virus Evolution



%====================
\textbf{Reviewer \#1}
%====================

Reviewer: 1

Comments to the Author
The manuscript by Carvalho et al., describes the analysis of foot-and-mouth disease viruses (FMDVs) of two different serotypes (O and A) based on partial genome sequences from samples collected during a 55 year period for serotype A and 16 years for serotype O. 
The sequence analysis data is linked to studies on the trade of FMDV susceptible animals, e.g. cattle and pigs. 
The study has some interest but from my point of view, there seem to be some surprising omissions of information which may, or may not, affect the conclusions that can be drawn.

Specific points
1)      In the Abstract, the text indicates that serotype O emerged (in South America) in around 1990. 
This is an odd statement to make since there are well known FMDV serotype O strains that predate this, e.g. O11 Campos (from Brazil in 1958), O1 Argentina (c. 1965) and O/M11/MEX from Mexico in 1952 which have all been sequenced in part and accession numbers are available (cited in Wright et al., (2013) Infect Genetics Evolution 20, 230-238).  
How does consideration of such sequences influence the information about the date of introduction of the viruses and their circulation in South America? 
In the Introduction, the authors indicate that: ?Historically, serotype O has been the most prevalent serotype on the continent?? (lines 53-55, P1). 
The authors need to explain why the earlier strains of FMD virus were not included in their analyses.

\begin{reply}
We have made the following change in the text:
\begin{quote}
\myeditThisIsAnEdit
\end{quote}
\end{reply}

2)      It is curious that there is no mention of serotype C FMDV in South America.

\begin{reply}

\end{reply}

3)      The sequence analysis is based on the VP1 coding region, this only represents about 630 nt out of a complete FMDV genome of about 8400nt. 
This information is not presented within the Introduction or Results section of the manuscript and only becomes apparent from Figure legends and Material \& Methods (i.e. it is quite well hidden). 
The resolution of the analyses based on VP1 coding region sequence information alone is necessarily limited. 
Full genome (or near full genome) sequencing can give much higher resolution, e.g. to the level of farm-to-farm spread (e.g. see Valdazo-Gonzalez et al., (2012) PLosOne 7(11) e49650).

\begin{reply}

\end{reply}

4)      The nature of the samples used for the virus sequence determination is also not indicated, have some been extensively passaged in cell culture? 
This information should be available using the accession numbers of the published sequences and clearly can influence the outcome of sequence comparisons.

\begin{reply}

\end{reply}

5)      I am not familiar with the ?root-to-tip? plots shown in Figure S1 but it seems to me that the slope of the line for serotype A over the period from 1995 to 2010 is not very different from that of serotype O over this limited time period and is rather different from that for the whole time period for serotype A. 
It would be useful if the authors commented.

\begin{reply}

\end{reply}

6)      A major concern about identifying origins of samples is having adequate coverage of samples from potential sources. 
It is not entirely clear to me that the coverage of FMDV strains circulating in South America is sufficient to be able to draw good conclusions. 
It may be that it is but this is not clearly demonstrated.

\begin{reply}

\end{reply}

7)      On P.4, lines 48-50. The text indicates that the importance of long range migration routes seems to differ for the two serotypes but the confidence values for the two serotypes seem to overlap extensively, so is this a real difference?

\begin{reply}

\end{reply}

8)      Recombination within the coding region for VP1 alone is rather rare and thus I am not sure the check for recombination was very justified or useful (P. 9 lines 8-11).

\begin{reply}

\end{reply}


%====================
\textbf{Reviewer \#2}
%====================

Reviewer: 2

Comments to the Author
The ?Spatio-temporal dynamics of FMDV in South America? study by Carvalho et al. describes the historical spatio-temporal dispersal of FMDV across the South America continent reconstructed using phylogeography analyses employed through a Bayesian spatial diffusion model. 
The reported results define transmission networks and transmission hubs (at country level) that would explain the historical spread of FMD within the continent. 
In addition, the authors deal with variables likely associated with the FMD spread (i.e. geographical distance, livestock density, and livestock trade) in trying to explain their causative effect associated with historical FMD outbreaks. 
As last attempt, the authors correlated the demographic dynamics of FMDV with reported number of FMD outbreak and vaccination coverage in trying to assess the impact of FMD control policies on the FMDV diversity and its population expansion/contraction through time. 
The paper has been already published as an arXiv (http://arxiv.org/abs/1505.01105) the 5th of May. 
Although the methodological approach has been previously used in different setting and the results would be interesting for a computational basis, there are several aspects of the study that need to be carefully considered before the paper would be suitable for publication. 
In fact, the presence of bias in the data used is potentially producing an incorrect picture of FMD in South America. 
In addition, although the authors are examining the potential impact of the sampling bias in the sequence data analysed, this is only properly discussed as Supplementary Text and not in the main paper, where they assume the results as correct, valid and without bias. 
One major problem of the study is the data. 
They claim to have analysed all the data publicly available in GenBank but (as detailed below) this is not correct and the analyses should be repeated including the full sequence dataset. 
I would be, therefore, really cautious to draw important conclusion from this study given the issues reported, which might hold true only for the time-frame pictured from the data you have analysed. 
As already said, this study need a proper revision before being published and this main revision would involve the re-analysis of all the data adding all the sequences available in GenBank and which are not included in this version.

\begin{reply}

\end{reply}

Problems of the study:
-       The authors claim to have used all publicly available VP1 sequences from GenBank, but after inspection this is not true. 
In fact, there are quite a number of sequences that has not been included in the study. If this has been done intentionally, the reason for this decision should be discussed in the paper; if not, I strongly suggest checking better in GenBank what is missing from your analyses (you have even the GenBank Accession Nos of the missing data in one of your reference [Malirat et al., 2007]). 
For the serotype O, for example, you are totally ?ignoring? the sequences before the 1994 but, what about the O/Campos (O/Br/58) and the Argentinian samples of the ?82-83 or the Caseros/67, the Selab/77? I could make the list much longer. 
This is valid for the serotype A as well (e.g. among others, where is the A10/Arg/61?). 
Therefore, if you want to claim to have analysed the complete VP1 coding sequence data for South America you need to re-perform all the analyses, because the results might provide you a completely different picture of FMD in South America (see your conclusion on the Colombian origin of the type O in South America).

\begin{reply}

\end{reply}

-       All the results discussed on the spatial diffusion of FMDV in the whole South America continent should be treated with cautions (potentially providing you incorrect data), considering that you have: missing information from missing sequences; a sampling bias in your data according to time and country. 
It is well known that FMDV was introduced (as you pointed out in the introduction) by human migration from Europe in the end of the 19th Century with early reports in Argentina between 1860 and 1870, and 1895 in Brazil. 
There were at least two distinct introductions in the North and one in the South but, before the 1922, it is really difficult to say which serotype was (i.e. before the FMDV typing was performed). 
However, it is clear that the early spread of FMD was coming from the South. 
Argentina at the time was one of the main export hubs of livestock to the continent and even the Mexico outbreak in 1922 has been attributed by the introduction of infected cattle from Argentina. 
This holds true for: Chile, 1920 outbreak (decline of cattle industry during 1912 in Chile with large introduction from Argentina); official report in Venezuela 1950 (potential from importation of Argentinian meats/livestock back to the 1947). 
It might worth to know that the Andes acted as a barrier for taking FMD out of Chile and the western part of South America, until when the regional animal movements and trade primarily caused the spread. 
The countries of the Rio de la Plata, which were sharing the Pampas ecosystem, experienced an early wave of disease spread and by the 1920 FMD was in Uruguay, Paraguay and Brazil. 
Historical data suggest that type O was introduced most likely from the South (maybe Argentina) and type A introduced from Europe. 
From your analysis the initial historical FMD wave has not been characterised (in the years before 1994 for type A; very limited and potentially biased before 1965 for type O). 
The only part which might be more realistic is the FMD transboundary movements within the ?countries triangle? of Colombia, Ecuador and Venezuela, that could sounds more like from Argentina-Venezuela-Colombia-Ecuador, even though you have the effect of sampling bias that needs to be discussed.

\begin{reply}

\end{reply}

-       Although you presented some data on sampling bias in your Supplementary Text (but this might be not satisfactory enough given the problem in the dataset), there is a real problem of sampling bias and this need to be addressed in your main text as well. 
How does the model deal with missing links? 
Is the prediction robust enough to provide a clear indication of virus spread in such a large geographical range? 
This could be a serious limitation (and problem) of your study. For serotype A, you analysed 131 VP1 sequences of which 44\% are from Argentina (of which $\sim$70\% are from 2000 and 2001) and 21\% from Venezuela (of which $\sim$86\% are recent samples - after 2001). 
In addition, the majority of your oldest samples are only from Brazil and Argentina. For serotype O, you have 167 sequences in total of which $\sim$54\% are from Ecuador (all after the 2002 and have been previously analysed - along with 30 sequences that have been included in this manuscript as well). 
Therefore, you have 90+30=120 sequences already analysed in a previous paper. 
Among the other, 36 sequences from Colombia ($\sim$22\% of the total) are barely covering the 2000s (as you claim 1994 to 2008), since you have 5 sequences from 2000, 1 from 2002 and 2 from 2008, a gap of 6 year. 
For the type A database, your oldest samples are only from Colombia. You attempted a random sub-sampling that, as far as I understood, have not taken into account the time of sampling, but just the quantity of data from each country. 
Maybe you need to account for time in your sub-sampling.

\begin{reply}

\end{reply}

-       When doing analysis on sequences extracted from GenBank a detailed list of the sequences with their GenBank Accession No (along with associated metadata) should always be provided. 
Although a webpage (but this is difficult to check and, probably, the majority of the readers would not bother to access to your website) has been set up for the paper there are no GenBank references for the serotype A. 
This information should be included either as a table in the main text or as a S3.

\begin{reply}

\end{reply}

-       This paper has been already published as an arXiv (http://arxiv.org/abs/1505.01105) which has been submitted the 5th of May and updated the 2nd of June (I received this review the 12th of June). 
Although it is common for theoretical maths, physics and computer sciences studies to be published as arXiv before being properly peer-reviewed, this is not the case with study dealing with topics as in this case. 
Since the study needs a substantial review and re-analysis of the data, I would suggest to withdraw your submission to arXiv

\begin{reply}

\end{reply}

Major Comments:
-       Page 1 Line 36: ?Our dating??. 
This result is only compatible with your dataset and must not be related with the incursion of FMD in South America. 
Your dating is referred to the MRCAs of the data you have analysed but not the MRCAs of both the type A and O clades in South America. 
As already detailed the occurrences of both serotypes are much earlier than your estimates, which therefore are misleading in the description. 
If you comment on the South America FMD phylogenetic history, you need to do that only in line with the data you analysed and not as a general picture.

\begin{reply}

\end{reply}

-       Page 2 Line 7: ?By the 1970s??. 
Again this is not true. 
During 1950s FMD was already causing problems in Argentina, Brazil, Chile, Peru, Uruguay, Venezuela, Colombia, and Ecuador. 
This picture is larger than a regional scale.

\begin{reply}

\end{reply}

-       Page 2 Line 33: ?using all??. 
You are not using all the sequences available in GenBank. 
For example and as already commented, I cannot find the O Campos in your fasta file (and this is only one). 
I strongly suggest doing a better search and re-perform all the analyses.

\begin{reply}

\end{reply}

-       Page 3 Line 20: ?the time of the most recent??. 
You need to clearly state here that these estimates hold true only for your sequences analysed (MRCSs of the data) and not of the entire South America because saying that is misleading and incorrect.

\begin{reply}

\end{reply}

-       Page 3 Line 21: ?indicating a more recent origin??. 
Again this is only true for your data and should be stated. 
Serotype O outbreaks have been reported in South America since the initial wave, but clear reports start from 1940-50 (e.g. massive outbreak in Peru in 1962; 1950 official report in Venezuela; 1957 A, O and C in the entire Rio Grande do Sul).

\begin{reply}

\end{reply}

-       Page 3 Line 22: the results show a faster clock for the type O than the A (even this is not really a lot faster - considering the VP1 only there is a difference between the two of $\sim$4nt changes per year). 
Might this be due to the different molecular clock model used for type A and O?

\begin{reply}

\end{reply}

-       Page 4 Line 49: ?Remarkably??. 
You claim that there is a difference between long-range migration routes between serotypes (reported as 0.14 and 0.05 for type A and O, respectively). 
However, the 95\% intervals are really similar and both containing the zero, I should then say that this is not so remarkable.

\begin{reply}

\end{reply}

-       Page 4 Line 57: ?The most probable??. 
This result might indicate that the 2001 reappearance of FMDV in Argentina was a persistent virus foci (maybe carrier?)?or maybe some missing links (i.e. sampling bias) exists in your data which are not including contemporary isolates from neighbouring countries (besides Brazil) and, therefore, this would impact on your results. 
Since this would be quite an interest topic (even though only on a retrospective line), you need to discuss this in more details, assessing as well the validity of your results.

\begin{reply}

\end{reply}

-       Page 5 results on the Venezuelan origin of Andean FMDV spread: 
Considering that you have a bias in your sequences for type A, you need to really consider with caution your results and discuss more about the impact it might cause.

\begin{reply}

\end{reply}

-       Page 5 Line 16: ?Similar to what was found for Venezuela??. 
You present data on Colombia saying that this results is similar to type A for Venezuela?however, I would rather imagine that you are discussing about source of type O in the north from Colombia. 
Is this true? If so, please rewrite the sentence to make that clear. 
In addition, since you have all the oldest historical samples of type O from Colombia, the logical reasoning would be that of course the analysis point to Colombia as the main transmission hub. 
But, what about the sequences you have not included in the analysis? 
Should this provide you a different picture? 
You need to clearly discuss this issue (and of course re-perform the analysed including all the data available)

\begin{reply}

\end{reply}

-       Page 5 Lines 34-38 and following paragraph (Lines 41-57): ?For serotype A??. 
This sentence is really confusing and need to be better formulated. 
For type A you find that geographical distance drives the diffusion, whilst this has a higher statistical support for type O but not like the cattle exchange. 
Now, the question is, what cattle exchange means? 
This implies geographic distance as well (because you are defining trade between countries, which in South America are not so very close), isn?t it? 
So, the geographic distance is the main effect of FMDV diffusion or a confounding effect for cattle trade? 
I am really struggling to find a logic behind this results (or its analytical approach) considering that you have a strong bias in your data (both spatially and temporally) and you are analysing the geographical distance and trade (both cattle and swine) variables separately?have you checked for multicollinearity?

\begin{reply}

\end{reply}

-       Page 6 Line 9: Sensitivity analysis. 
The sub-samples (as referred in table S4 and S5) is excluding the over-represented countries, i.e. Argentina and Colombia. 
However, if you exclude Argentina from the type A data, you have now Venezuela that is over-represented (the same holds true for type O, for which Colombia is the over-represented after the exclusion of Ecuador). 
Since both the Argentinian and Ecuador samples are, let?s say, monophyletic (collected for the majority within epidemics), this are not really changing the global picture. 
In addition, you claim (Page 24 Line 37) that removing Argentina from the type A analysis move the MRCA estimate of $\sim$6 years. Is this because you eliminate one of the oldest sequences present in your data, thus leaving only the Brazil ?58 and, therefore, introduce a more substantial sampling bias/uncertainty? 
For type O, considering that the oldest samples are from Colombia, removing Ecuador has no impact in the results. 
I am getting confused to understand which methodology is behind your random sampling approach used for the 5 sub-sampling. Is this a proportional random sampling (with a temporal sub-sampling as well) of each country?

\begin{reply}

\end{reply}

-       Page 6 Demographic reconstruction: You are discussing the increase/decrease in FMDV diversity according to the reported activity of FMD and the control policies (i.e. vaccination) imposed. 
However, it seems that it is difficult to correlated like-with-like in your graph(s): you have doses/head of vaccine (this could be monovalent, bi-, tri-; strain(s) used), no of FMD cases (I suppose reported no of outbreak - this could be 1 individual of 1000s of animals infected) and viral diversity. 
One point that is completely missed in your discussion is the vaccine efficacy and this might impact in your analysis (i.e. some reports of drop in efficacy of the O campos vaccine). 
In addition, you comment that after 2001 an increase in vaccine doses resulted in a decrease in viral diversity: although from the FMD outbreak data is true for type A, it is not clearly valid for type O, which maintained a more stable trend (of course with some fluctuations). 
Is this, again, an issue due to bias in your data? A previous study (de Silva et al., 2012) describes how BSP incorrectly reconstructed a decrease in the last part of a datum epidemic when the population was still growing. 
This problem was related to the lack of genealogical information at later times. Would this be the case for your analysis as well?

\begin{reply}

\end{reply}

-       Page 7 Line 31: ?the inclusion of archival??. 
You are discussing about the impact of using an outgroup into your analysis and fail to analyse that (although you could easily extract some sequences from GenBank). 
In addition, you claim that the type O was circulating in the continent with its root in Colombia but you do not include any samples prior to the 1994. 
This analysis is incorrect and should be appropriately revised.

\begin{reply}

\end{reply}

-       Page 7 Line 55: ?Previous studies??. 
It seems that the study you referred describes similar cycle of 4-5 years for both serotypes and, moreover, this would be really complicated and dangerous to apply as a general rule (since it is a country-based estimate). 
I would suggest deleting this sentence. In addition, from you skyride plot it is difficult to say that a 4-5 year FMDV cycles exists.

\begin{reply}

\end{reply}

-       Page 8 Line 8: ?The diversity bottleneck??. 
You commented about the bottleneck for the type A diversity reconstruction as an effect of FMD epidemics affecting several countries after the 2000, but I would remind you that the majority of your samples (collected mainly from epidemics) are from the 2000 afterwards. 
Therefore, this again would be a confounding effect due to bias in your data (i.e. is the skyride estimate affected by the number of coalescent events in your phylogeny?).

\begin{reply}

\end{reply}

-       Page 8 Lines 18-21: ?Our results suggest??. 
This is incorrect and needs to be properly assessed when a more comprehensive analysis, which would include all the type O isolates, has been performed. 
If the paper of Carvalho et al., 2013, indicates the same results (i.e. describing the origin of the FMDV serotype O in South America from Colombia using the very same data), that needs a proper review as well.

\begin{reply}

\end{reply}

-       Page 8 Line 26: ??viral effective size??. 
This is a reminder about the previous comment on Demographic reconstruction.

\begin{reply}

\end{reply}

-       I am not familiar with the methodology behind that, but I suppose that the analysis of epidemiological predictors (i.e. cattle and pig trade/livestock data) seems to have been constructed around an ?average? value which potentially does not describe the space-time trends of trade routes and animal movements. 
Does this have an impact on your results generated? 
If so, what is the validity of those results? Please, comment on this.

\begin{reply}

\end{reply}

-       You might consider using some sequences (e.g. O BFS 1860) as outgroup for your phylogenetic reconstruction and perform a more detailed analysis that would include all your sequences (maybe using a random local clock to account for variability in the rates), therefore shaping the entire tree topology. 
In addition, the phylogenetic trees, as are presented now, are confusing and really difficult to read.

\begin{reply}

\end{reply}

Minor Comments:
-       Page 1 Line 34: ?environmental??. 
Are you really using environmental data (e.g. air and bathing water quality)? 
Or do you mean livestock population and trade data, so more epidemiologically-related data or population data?

\begin{reply}

\end{reply}

-       Page 1 Line 42: ?Our findings??. 
This is a general sentence and might lead to the assumption that evolutionary and spatial dynamics of serotype A and O are globally different (which might be not the case). 
Just highlight the South America setting.

\begin{reply}

\end{reply}

-       Page 1 Line 50: ?, the most important??. 
Is FMD the most important animal disease or, better, is one of the most?

\begin{reply}

\end{reply}

-       Page 2 Line 17: References 4 and 14 are duplicates.

\begin{reply}

\end{reply}

-       Page 2 Line 17: Use Di Nardo et al. [12].

\begin{reply}

\end{reply}

-       Page 2 Line 18 and Line 20: ?environmental??. 
Check the meaning of ?environmental data? with what you are trying to analyse.

\begin{reply}

\end{reply}

-       Page 2 Line 28: ??in the continent.?. 
Which one? 
Do you mean at ?continental? level? 
Or you are only referring to South America?

\begin{reply}

\end{reply}

-       Page 3 Line 48: ?..we employ an asymmetric??. 
You have already detailed your analysis procedures in the Material and Methods section. 
This could be deleted.

\begin{reply}

\end{reply}

-       Page 5 Line 12: ?We provide evidence of Venezuela..?. 
Which region you are referring to? 
The Andean region? 
Please, specify.

\begin{reply}

\end{reply}

-       Page 5 Line 26: ??trade and viral diffusion, we collected??. 
I think it would be better to say ?we obtained data?from?? because maybe you have not been in the field collecting data.

\begin{reply}

\end{reply}

-       Page 7 Line 22: I would rather use: ?see Figure 3 in [6]? or ?see FMD historical outbreak data in [6]?.

\begin{reply}

\end{reply}

-       Page 8 Line 7: ?..for viral Ne in both??. 
You previously discuss about viral diversity and now present the effective population size (Ne). 
Since the skyline family is based on the ?=Net, you should describe Ne only if you have extracted that estimate with the appropriate measure of generation time, which you are not having or even discussing (i.e. in your graph you should report in your y-axis legend the compound value as well - Net). 
I suggest using viral diversity throughout the paper.

\begin{reply}

\end{reply}

-       Page 9 Line 13: It seems you used BEAST 1.7.5 - or even and older 1.7.2 version - (from your web available .xml) to perform the analyses, although a more recent version is available (1.8.2). 
Is the latest version more robust and efficient in the results generated? 
Have you re-analysed your data using the latest version and producing similar results?

\begin{reply}

\end{reply}

-       You have a type O sequence from Peru (GenBank Accession No. HQ695844.1) you say collected in 2004, whilst in your previous paper on Ecuador is defined as 1994. 
I checked in GenBank and this is from 2004. 
Therefore, you need to amend your previous paper with the correct date.

\begin{reply}

\end{reply}


%====================
\textbf{Reviewer \#3}
%====================

Reviewer: 3

Comments to the Author
Your reviewer is not an expert in phylogeographic or phylodynamic analyses, but does have experience in Bayesian analyses in general and FMDV epidemiology and evolution.

The authors present an analysis of the spatio-temporal dynamics of FMDV in South America. 
They conclude that serotypes O and A behave quite differently, with different rates of evolution, different circulation networks, and different predictors of spread. 
Their analysis is based on $\sim$300 VP1 sequences retrieved from public databases, and combines a series of Bayesian phylogenetic/geographic/dynamic analyses to come to its conclusions.

It is unfortunate that there is such limited sequence data to work with, especially since it is only for VP1, a very small component of the FMDV genome (albeit a very variable one), but I am concerned more by other problems with the underlying data on which it is based, and the soundness of the resulting conclusions. 
Assuming these concerns can be addressed, however, the paper provides new conclusions on the spread of the disease in South America, will be of interest and value to the FMD research community, and should be published.

\begin{reply}

\end{reply}

This would be especially true is there were any evidence in terms of the known epidemiology of the disease which might support the assertion that it differs so strikingly between the serotypes, such as a differential likelihood in different serotypes of airborne transmission (which might favour geographical spread?) or subclinical infection and subsequent transmission (which might favour transmission despite inspection as a result of trade?).

\begin{reply}

\end{reply}

My specific major concern is with the effect of differential sampling effort resulting in different numbers of samples in each country, rather than this being a feature of the epidemiology of the disease. 
Chile, Guyana, French Guiana and Suriname have no FMDV sequences, and Paraguay has no A sequence even though wrlfmd.org shows that Paraguay, Guyana and Chile have experienced recorded outbreaks. 
It is very likely that French Guiana and Suriname have too. 
Peru, Paraguay and Uruguay also have very low numbers of samples in total. Some work has been carried out to investigate the sensitivity to spatial sampling heterogeneity, but not with respect to the predictors of spread. 
Depending on whether there is detailed information on the location of VP1 sequences within countries, this inference could depend strongly on these poorly sampled (or unsampled) countries, and it would seem important to investigate whether this effect alters the conclusions, perhaps by removing the poorly sampled countries (since we can't add in potential missing countries), and just investigating spread between the countries with high sample numbers for the serotype.

\begin{reply}

\end{reply}

It would also seem important to investigate whether the reason for the difference in observed predictors relates not to the different epidemiology of the serotypes, but to the different control policies in the different time periods studied. 
This could be investigated by doing all of the inference for serotype A again based just on the recent data.

\begin{reply}

\end{reply}

Also, while it seems plausible to suppose that there might be a different substitution rate, this difference seems high, and a casual inspection of Figure S1 also suggests that there might be much faster substitution rate in A during the period for which O data exists, though I don't know why that might be the case.

\begin{reply}

\end{reply}

Finally, I see no explanation or justification for why the molecular clock model should be different between two serotypes of the same virus circulating through the same species in the same region. 
Some explanation would seem appropriate, or an investigation of what the implications for other conclusions might be if this might not be the case.

\begin{reply}

\end{reply}

Minor detail:

Kullback-Leibler is occasionally misspelt as Kullback-Liebler.

\begin{reply}

\end{reply}



\clearpage

\bibliography{baele}
 
\end{letter}




\end{document}
